app/assets/javascripts/posts.coffee

```diff
@@ -0,0 +1,36 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+
+# Get the relevant posts and insert them in the "relevant to you" section
+Spaces.updateRelevantPosts = () ->
```

Comment:
instead of sticking the methods here (and having then globally-scoped) I think it would be better to create a new `Spaces.PostsController` that you instantiate in the view (like with the ajax forms) that provides these methods and hooks them into the DOM